tasks/swap.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Check if swapfile exists
+  stat:
+    path={{ swap_file }}
+    get_checksum=False
+    get_md5=False
+  register: swapfile
+
+- name: Enable swapfile with length {{ swap_size }}
+  command: "{{ item }}"
+  with_items:
+    - fallocate -l {{ swap_size }} {{ swap_file }}
```

Comment:
split those 3 commands in 3 tasks

last is - use `swapon -a` (after you added it in fstab)